src/compiler/output-targets/dist-collection/index.ts

```diff
@@ -30,7 +32,22 @@ export const outputCollection = async (
           outputTargets.map(async (o) => {
             const relPath = relative(config.srcDir, mod.jsFilePath);
             const filePath = join(o.collectionDir, relPath);
-            await compilerCtx.fs.writeFile(filePath, code, { outputTargetType: o.type });
+
+            // Transpile the already transpiled modules to apply
+            // a transformer to convert aliased import paths to relative paths
+            const { outputText } = ts.transpileModule(code, {
```

Comment:
Only downside to this approach is, for this target, we'll essentially end up transpiling each module twice. However, I ran a couple builds with this change and without and only seemed to affect Stencil build times within a project by a second or so (that testing was done with the framework code base)